### PR TITLE
chore(react-chart): title and legend for demo

### PR DESCRIPTION
**Pull request by DmitryBogomolov**
Adds "real" data, title and legend for [this demo](https://devexpress.github.io/devextreme-reactive/react/chart/demos/more_features/zoom-and-pan/).

### CODE REVIEW

File: packages/dx-react-chart-demos/src/demo-sources/zoom-pan/scatter.partial.jsxt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      <table>
+        <tr>
+          <td><TooltipCell>Life Expectancy</TooltipCell></td>
+          <td><TooltipCell>{arg}</TooltipCell></td>
+        </tr>
+        <tr>
+          <td><TooltipCell>Birth Rate</TooltipCell></td>
+          <td><TooltipCell>{val}</TooltipCell></td>
+        </tr>
+      </table>
```
Comment (Krijovnick) [inline packages/dx-react-chart-demos/src/demo-sources/zoom-pan/scatter.partial.jsxt]: Maybe make tooltip design like in the [devextreme charts demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/Overview/jQuery/Light/): with bold text in the header and add opacity to the splitter, `Life Expectancy` and `Birth Rate` phrases?

File: packages/dx-react-chart-demos/src/demo-sources/zoom-pan/bootstrap4/scatter.partial.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+// BLOCK:body
+const TooltipCell = props => (
+  <span {...props} />
+);
+const TooltipSplitter = ({ className, ...props }) => (
+  <hr className={classnames('mt-1 mb-1', className)} {...props} />
+);
+const TooltipHeader = props => (
+  <h6 {...props} /> // eslint-disable-line jsx-a11y/heading-has-content
```
Comment (Krijovnick) [inline packages/dx-react-chart-demos/src/demo-sources/zoom-pan/bootstrap4/scatter.partial.jsx]: Is there comment required?

Comment (DmitryBogomolov) [reply packages/dx-react-chart-demos/src/demo-sources/zoom-pan/bootstrap4/scatter.partial.jsx]: The suppressed error is the following:
> Headings must have content and the content must be accessible by a screen reader

**Comment by MaximKudriavtsev** [comment]
image
image

The values are mixed up

File: packages/dx-react-chart-demos/src/demo-sources/zoom-pan/scatter.partial.jsxt (hunk 1 above)
Comment (DmitryBogomolov) [reply packages/dx-react-chart-demos/src/demo-sources/zoom-pan/scatter.partial.jsxt]: Here is how it looks now.
pr2058_bs4
pr2058_mui

Headers visually are not so bold as those in *devextreme demo* but they have `font-weight: 500` like in *devextreme demo*. With `600` they would look rather ugly.

----
The amount of non-similarity between *bs4* and *mui* markup is TERRIBLE though.

**Comment by MaximKudriavtsev** [comment]
BTW, can we write the scale titles? This point will be helpful to understand these data set.

**Comment by DmitryBogomolov** [comment]
No, there is no builtin mechanism for axis title.
And adding it via overriding **Axis.rootComponent** does not seem an easy option by now.

**Pull request merged**